who_l3_smart_tools/cli/indicator_testing.py

```diff
@@ -3,9 +3,9 @@
 import argparse
 from pathlib import Path
 import pandas as pd
-from who_l3_smart_tools.core.indicator_testing.bundle_generator import BundleGenerator
-from who_l3_smart_tools.core.indicator_testing.data_generator import DataGenerator
-from who_l3_smart_tools.core.indicator_testing.scaffolding_generator import (
+from ..core.indicator_testing.bundle_generator import BundleGenerator
```

Comment:
I would lean towards absolute imports. These can get confusing very fast. They are not as easy to read as the absolute ones.

Reply:
I don't disagree, but absolute imports require hacking the `sys.path()`, whereas relative imports do not, and in my mind, some ugly import `from`s are substantially better than path modifications.

Reply:
```
Traceback (most recent call last):
  File "/home/kenneth/Code/DiGi/who_l3_smart_tool/who_l3_smart_tools/cli/terminology.py", line 7, in <module>
    from ..core.terminology.who.terminology import HIVTerminology
ImportError: attempted relative import with no known parent package
```

That is the error I get when I run `who_l3_smart_tools/cli/terminology.py`  from this PR branch. This should not be an issue though if repo is installed as a pip package. In #19 , I removed the `sys.path()` hack and the commands work if installed as package. The error is because the repo root is not in python path.